Dockerfile

```
@@ -1,26 +1,27 @@
 FROM aiidateam/aiida-core:1.6.3
 
-LABEL maintainer="Materials Cloud Team <aiidalab@materialscloud.org>"
+LABEL maintainer="AiiDAlab Team <aiidalab@materialscloud.org>"
 
 # Configure environment.
 ENV AIIDALAB_HOME /home/${SYSTEM_USER}
 ENV AIIDALAB_APPS ${AIIDALAB_HOME}/apps
 ENV AIIDALAB_DEFAULT_GIT_BRANCH master
 
+ENV QE_APP_VERSION develop
+ENV AWB_APP_VERSION v1.0.0b18
```

This should be empty so that the base image does not install the app:
```suggestion
ENV AWB_APP_VERSION
```